server/build.gradle

```diff
@@ -35,13 +36,14 @@ dependencies {
     implementation "io.micronaut.sql:micronaut-jdbi"
     implementation "io.micronaut.data:micronaut-data-processor"
     implementation "io.micronaut:micronaut-http-client"
+    implementation "io.micronaut.rxjava3:micronaut-rxjava3"
```

Comment:
Micronaut team recommends using projectreactor instead:
> We recommend applications currently using RxJava2 to switch to Project Reactor because that will lead to fewer classes on the runtime classpath and fewer potential issues with context propagation and reactive type conversion.

https://micronaut.io/2021/08/18/micronaut-framework-3-released/

Can we do that?

Reply:
Not needed anymore